wsgi.py

```diff
@@ -0,0 +1,15 @@
+import os 
+
+from yoconfigurator.base import read_config
+
+app_dir = os.path.abspath(os.path.dirname(__file__))
+conf = read_config(app_dir)
+aconf = conf.lintreview
+cconf = conf.common
+
+activate_this = conf.deploy.root + '/lintreview/live/virtualenv/bin/activate_this.py'
+execfile(activate_this, dict(__file__=activate_this))
+
+os.environ.setdefault("LINTREVIEW_SETTINGS", conf.deploy.root + '/lintreview/live/settings.py')
+
+from lintreview.web import app as application
```

Comment:
E402 module level import not at top of file
F401 'lintreview.web.app as application' imported but unused